ERCS/draft_encrypted-types.md

```diff
@@ -0,0 +1,596 @@
+---
+eip: xxxx
+title: Standard for Encrypted Data Types, Validation, and Confidential Operations in Smart Contracts
```

Comment:
```suggestion
title: Encrypted Data
```
Standard shouldn't be used in an ERC title.
You may also want to use a shorter title (if this encompasses everything in the ERC).